.github/workflows/scorecard_action.yml

```diff
@@ -28,7 +28,7 @@ jobs:
           persist-credentials: false
 
       - name: "Run analysis"
-        uses: ossf/scorecard-action@c1aec4ac820532bab364f02a81873c555a0ba3a1
+        uses: ossf/scorecard-action@5c8bc69dc88b65c66584e07611df79d3579b0377
```

Comment:
I think this change is out of the scope of this PR, would you mind sending it by opening another PR, does it make sense?

Reply:
this had been automatically been added to my commit. my local branch has same value of upstream. Are github actions mutating this ?